chain/beacon/drand/drand.go

```diff
@@ -164,33 +174,28 @@ func (db *DrandBeacon) getCachedValue(round uint64) *types.BeaconEntry {
 	return v
 }
 
-func (db *DrandBeacon) VerifyEntry(curr types.BeaconEntry, prev types.BeaconEntry) error {
-	if prev.Round == 0 {
-		// TODO handle genesis better
-		return nil
-	}
-
-	if curr.Round != prev.Round+1 {
-		return xerrors.Errorf("invalid beacon entry: cur (%d) != prev (%d) + 1", curr.Round, prev.Round)
-	}
```

Comment:
It's safe to drop this for both "current" Drand and quicknet, right?

Reply:
I think so - it doesn't seems like in the current implementation the previous beacon should be cached, so there's no way for someone to e.g. provide a historical beacon that gets validated, though might be worth confirming

Reply:
hmm well actually we only check the `MaxBeaconRoundForEpoch` - perhaps it _is_ possible to provide a historical beacon without a check of the round?

Reply:
I think it is safe although it